### ISSUE TITLE
keeping mirrors up to date

### ISSUE THREAD

**Issue by rillian** [gcp/opus-tools#2]
We were talking about setting up more official mirrors of xiph.org repos here. Ideally we'd have an organization and a script to keep the up to date.

@xiph are you still using your account? Would you be willing to donate it to https://xiph.org/ ?

**Comment by gcp** [comment]
This is more my personal working copy than a mirror. Personally I hate it with a passion how github seems to intentionally hide upstream if it isn't a github repo itself. I wish there's be an option "[x] hit people who send pull requests to this repo with an upstream-clue hammer".

**Comment by rillian** [comment]
Fair enough. I think there's value in having mirrors here though, if only for discoverability. I created a xiph organisation today, so we 'just' need a commit hook to update copies here.

**Comment by dangerrangersa** [comment]
The account is now yours. Enjoy.

**Comment by rillian** [comment]
Many thanks, @dangerrangersa!
